crime-commons-spring-boot-starter-rest-client/src/test/java/uk/gov/justice/laa/crime/commons/config/RestClientAutoConfigurationTest.java

```diff
@@ -230,6 +232,19 @@ void givenRegistrationIdGetExchangeFilterWithPopulatesCorrectOAuthAttributes() {
                 });
     }
 
+    @Test
+    void traceIdHandlerIsConditionalOnTracerBean() {
+        this.contextRunner
+                .run((context) -> assertThat(context).doesNotHaveBean(TraceIdHandler.class));
+    }
+
+    @Test
+    void braveAutoConfigurerConfiguresTraceIdHandler() {
```

Comment:
Minor, but the `BraveAutoConfiguration` class just sets up the Tracer bean. It's the `RestClientAutoConfiguration` class that configures the `TraceIdHandler`. Therefore, I'd propose changing the name of the test to: `restApiClientConfigurerConfiguresTraceIdHandler`